### PR TITLE
Conversion class minutes to seconds

### PR DESCRIPTION
**Pull request by jean-farinea**
1. Conversion class created to convert minutes into seconds
2. Test class created for Conversion class

### CODE REVIEW

File: src/main/java/lesson3/task4/Conversion.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package lesson3.task4;
+
+public class Conversion {
```
Comment (strzeleckik) [inline src/main/java/lesson3/task4/Conversion.java]: In your potential system you can have many convertions: Currency convertion, Units convertions etc. You should name your classes in such a way that we will know exactly with what we are delaing with. Here in my opinion it should be sth like: TimeConverter

**Review by strzeleckik — CHANGES_REQUESTED** [review]
Small change

**Comment by jean-farinea** [comment]
Class refactored and renamed within test class, please check.

**Review by strzeleckik — APPROVED** [review]
ok

**Pull request merged**